### PR TITLE
Issue 10499 pdf thumbnailer

### PR DESCRIPTION
**Pull request by wezell**

### CODE REVIEW

**Comment by wezell** [comment]
@dsilvam @jtesser  I am ok leaving this out of 3.7 if you all are not comfortable with the changes.

File: dependencies.gradle [hunk 1]
```diff
@@ -262,9 +262,9 @@ dependencies {
 	compile group: 'com.twelvemonkeys.imageio', name: 'imageio-thumbsdb', version:'3.2.1'
 	compile group: 'com.twelvemonkeys.imageio', name: 'imageio-batik', version:'3.2.1'
 	compile group: 'com.twelvemonkeys.imageio', name: 'imageio-clippath', version:'3.2.1'
-	compile group: 'com.twelvemonkeys.servlet', name: 'servlet', version:'3.2.1'
-
-
+	//compile group: 'com.twelvemonkeys.servlet', name: 'servlet', version:'3.2.1'
+	compile group: 'org.apache.pdfbox', name: 'pdfbox', version: '2.0.1'
+	compile group: 'org.bouncycastle', name: 'bcprov-jdk15on', version: '1.56'
```
Comment (wezell) [inline dependencies.gradle]: Bouncy Castle's Provider is required to thumbnail encrypted PDFs.

File: src/com/dotmarketing/servlets/ShortyServlet.java [hunk 2]
```diff
@@ -71,8 +71,19 @@ private void _serve(final HttpServletRequest request, final HttpServletResponse
...
+    
+    StringTokenizer tokens = new StringTokenizer(uri, "/");
+    if (tokens.countTokens() < 2) {
+      response.sendError(404);
+      return;
+    }
 
+    tokens.nextToken();
+    String id = tokens.nextToken();
+    String fieldName = tokens.hasMoreTokens() ? tokens.nextToken() : NOT_FOUND;
```
Comment (wezell) [inline src/com/dotmarketing/servlets/ShortyServlet.java]: This fixes a bug whereby Shorty can't serve a non-defualt field that has a camelcase velocity variable

**Review by dsilvam — APPROVED** [review]

**Comment by dsilvam** [comment]
PR: https://github.com/dotCMS/core/pull/10506

**Pull request merged**